Software/src/src.pro

```diff
@@ -179,10 +179,23 @@ unix:!macx{
 
     QMAKE_CXXFLAGS += -std=c++11
     contains(DEFINES,PULSEAUDIO_SUPPORT) {
-        LIBS += -lpulse -lfftw3
-        SOURCES += PulseAudioSoundManager.cpp
-        HEADERS += PulseAudioSoundManager.hpp
-    }
+        INCLUDEPATH += $${PULSEAUDIO_INC_DIR} \
+            $${FFTW3_INC_DIR}
+
+        defined(PULSEAUDIO_LIB_DIR, var) {
+            LIBS += -L$${PULSEAUDIO_LIB_DIR}
+            QMAKE_POST_LINK += $(INSTALL_PROGRAM) \"$${PULSEAUDIO_LIB_DIR}/libpulse.so.0\" $(DESTDIR) $$escape_expand(\n\t)\
```

Comment:
Not too sure if there's a point copying libs.

Reply:
Having them there would imply that the destination path is in the `LD_LIBRARY_PATH`  which wouldn't be the case.

Why not check `PULSEAUDIO_INC_DIR` for existence too?

Reply:
Yeah, I'd remove the QMAKE_POST_LINK lines. If you're crazy enough to use non-system libs, you'd probably get it to build yourself.

> Why not check PULSEAUDIO_INC_DIR for existence too?
Maybe check /usr/include/pulse{,/pulseaudio.h}? Just adding undefined PULSEAUDIO_INC_DIR to INCLUDEPATH didn't seem to have ill effects. And if it is defined but invalid, well, the first point :D

Reply:
I agree. I don't think these are needed.